tests/unit/type/stream.tcl

```diff
@@ -435,6 +435,17 @@ start_server {
 
         # verify nil is still received when reading last entry
         assert_equal [r XREAD STREAMS lestream +] {}
+
+        # case when stream created empty
+
+        # make sure the stream is not initialized
+        r DEL lestream
+
+        # create empty stream with XGROUP CREATE
+        r XGROUP CREATE lestream legroup $ MKSTREAM
+
+        # verify nil is received when reading last entry
+        assert_equal [r XREAD STREAMS lestream +] {}
```

Comment:
does this mean that this change would be a break change?
before this change `XREAD` retures `the stream last element ID is 0-0`, but now returns an empty array.

Reply:
I wouldn't classify it as breaking change, because it is a bugfix. The error shoudn't have happened in your case, stream is just empty.
```
127.0.0.1:6379> DEL mystream
(integer) 0
127.0.0.1:6379> XGROUP CREATE mystream mygroup $ MKSTREAM
OK
127.0.0.1:6379> XREAD COUNT 1 STREAMS mystream 0
(nil)
127.0.0.1:6379> XREAD COUNT 1 STREAMS mystream +
(error) ERR the stream last element ID is 0-0
```
In this example reading from the beginning returns `(nil)`  but reading from the end fails with error (inconsistency). btw error message is wrong too, there are no elements in this stream yet